src/oge/data_cleaning.py

```diff
@@ -1002,7 +1018,8 @@ def clean_cems(year: int, small: bool, primary_fuel_table, subplant_emission_fac
     )
 
     # manually remove steam-only units
-    cems = manually_remove_steam_units(cems)
+    # NOTE(greg): disabling this for the 2022 data release
+    # cems = manually_remove_steam_units(cems)
```

Comment:
Why do we stop removing these units in 2022?

Reply:
Sorry this is unclear! This is not just for 2022, we are stopping using this method altogether. See note about this in the main PR description. I'll delete this whole line.

Reply:
I understand it will apply to all years from now on. My question is why do we stop removing steam-only units. Do we feel confident estimating emissions from these generators moving forward?